Packs/Base/ReleaseNotes/1_6_8.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CommonServerPython
+- General performance and reliability improvements.
```

Comment:
Add proper release notes. Explain that we moved IAM classes into separate module